services/src/main/java/io/meeds/gamification/rest/builder/RuleBuilder.java

```diff
@@ -220,10 +219,6 @@ public static UserInfoContext toUserContext(ProgramService programService,
     return userContext;
   }
 
-  public static String processRichEditorContent(XMLProcessor xmlProcessor, String content) {
```

Comment:
I think we may have a regression on Email Notification content, can you double check please. In addition, can you check please if an XSS attack is possible when removing this.
This change was introduced using the commit https://github.com/Meeds-io/gamification/commit/3184ce8c2de08792c642ef2b815241f201b67a2c

As described:
> This change will transform Action descriptions before retrieving to UI to auto determine the links targets.

Can you ensure that the links targets are ok switch the needed behavior (see https://builders.meeds.io/portal/meeds/tasks/taskDetail/2264)

Reply:
All the scenarios I did were good for me (same behavior on ACC) @boubaker